### ISSUE TITLE
Helm chart cant build ingress

### ISSUE THREAD

**Issue by KulykDmytro** [MarquezProject/marquez#1090]
When trying to build helm chart with `ingress.enabled: true`
Helm is throwing an error:
> Error: template: marquez/templates/ingress.yaml:35:30: executing "marquez/templates/ingress.yaml" at <include "common.names.fullname" .>: error calling include: template: marquez/charts/common/templates/_names.tpl:22:14: executing "common.names.fullname" at <.Values.fullnameOverride>: can't evaluate field Values in type string

caused by tpl usage in scope of range context (which cant access toplevel attributes)

**Comment by wslulciuc** [comment]
Thanks @KulykDmytro for giving our chat a try. There have been recent changes to the chart, and this may not have been caught during general testing of the chart. I'll look into this bug this week and hopefully have a fix by our next release (`0.13.0`)

**Comment by KulykDmytro** [comment]
```
diff --git a/chart/templates/ingress.yaml b/chart/templates/ingress.yaml
index e68eb57..c361259 100644
--- a/chart/templates/ingress.yaml
+++ b/chart/templates/ingress.yaml
@@ -32,11 +32,11 @@
         paths:
           - path: /api/
             backend:
-              serviceName: {{ include "common.names.fullname" . }}
+              serviceName: {{ include "common.names.fullname" $ }}
               servicePort: http
           - path: /
             backend:
-              serviceName: {{ include "common.names.fullname" . }}-web
+              serviceName: {{ include "common.names.fullname" $ }}-web
               servicePort: http
   {{- end }}
 {{- end }}
```

**Comment by wslulciuc** [comment]
Easy enough! Did you want to open a PR for the fix?

**Comment by KulykDmytro** [comment]
I'm pretty new with contribution, however will try

**Comment by KulykDmytro** [comment]
fixing #1090

**Comment by wslulciuc** [comment]
Fixed #1094
